app/Repositories/UserRepositoryInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Gladiator\Repositories;
+
+interface UserRepositoryInterface
```

Comment:
This is such a nitpick, but Taylor seems to like calling these "Contracts" in the Laravel world.